Content/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM microsoft/dotnet:2.2-aspnetcore-runtime-alpine
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.0-alpine
 COPY /deploy /
 WORKDIR /Server
 EXPOSE 8085
-ENTRYPOINT [ "dotnet", "Server.dll" ]
\ No newline at end of file
+ENTRYPOINT [ "./Server" ]
```

Comment:
why this change? I get following error: `standard_init_linux.go:211: exec user process caused "exec format error"`

Reply:
.NET Core 3 should produce executables by default, so I thought this would be the way to launch it now.

Reply:
Or actually, I get both now. I suppose this line may be reverted then.